tests/Hyperwallet/Tests/HyperwalletTest.php

```diff
@@ -3535,6 +3535,56 @@ private function createAndInjectApiClientMock(Hyperwallet $client) {
         return $apiClientMock;
     }
 
+    //--------------------------------------
+    // Document upload for users
+    //--------------------------------------
+    public function testuploadDocumentsForUser_withoutUserToken() {
+        // Setup
+        $client = new Hyperwallet('test-username', 'test-password');
+
+        try {
+            $client->uploadDocumentsForUser('', null);
+            $this->fail('HyperwalletArgumentException expected');
+        } catch (HyperwalletArgumentException $e) {
+            $this->assertEquals('userToken is required!', $e->getMessage());
+        }
+    }
+
+    public function testuploadDocumentsForUser() {
+        // Setup
+        $client = new Hyperwallet('test-username', 'test-password');
+        $apiClientMock = $this->createAndInjectApiClientMock($client);
+        $userToken = "user-tokenn";
+
+        $options = array(
+            'multipart' => [
+                [
+                    'name'     => 'data',
+                    'contents' => '{"documents":[{"type":"DRIVERS_LICENSE","country":"AL","category":"IDENTIFICATION"}]}'
+                ],
+                [
+                    'name'     => 'drivers_license_front',
+                    'contents' => fopen('/Users/ramahalingam/Desktop/L1.png', "r")
+                ],
```

Comment:
This looks like a path on your machine. Can we change/remove this?

Reply:
Please put the png file in a relative path within the test folder.

Reply:
Added the png files to the resources folder and changed the test accordingly.